### ISSUE TITLE
Type definitions for sync API

### ISSUE THREAD

**Issue by voodoo11** [adaltas/node-csv-generate#10]
Hi,
Thanks for great CSV parsing library!

Any chances you could add also Typescript type definitions to sync version? (`sync.d.ts` file)

**Comment by wdavidw** [comment]
Of course, would you have time to contribute it? I'm overbooked at the moment.

**Comment by maxfriedmann** [comment]
I think this issue is partly done already, at least I can import parse, transform and stringify via ES6 imports in csv/lib/sync.

Any reason why `//import * as generate from "csv-generate/lib/sync";` is commented out?

**Comment by wdavidw** [comment]
Where did you find the sync declaration ? Look at the [lib folder](https://github.com/adaltas/node-csv-generate/tree/master/lib), I can't find any reference to the commented import statement.

**Comment by maxfriedmann** [comment]
in the csv/lib/sync.d.ts, am in the wrong repo here I guess?

![image](https://user-images.githubusercontent.com/2809483/78116104-1b019080-7404-11ea-9f67-3ec462b5caa4.png)

**Comment by wdavidw** [comment]
the `csv` package is a sort of wrapper beind the 4 packages you see in your screenshot. `generate` is commented because it doesn't exists, thus this issue. The link sent above send you to the right location where the definition shall land.
